mixer/pkg/il/interpreter/interpreterRun.go

```diff
@@ -36,8 +36,10 @@ func (in *Interpreter) run(fn *il.Function, bag attribute.Bag, step bool) (Resul
 	var t2 uint32
 	var t3 uint32
 	var ti64 int64
+	var ti642 int64
```

Comment:
Can you run the IL interpreter bench to compare before-after numbers for this change? It should be in the same folder. This piece of code has some Golang implementation specific optimizations, so adding new locals can potentially impact perf.

There are comments in the file with previous run values, which I am 99.9% sure is stale at this point